modules/pbm/bitmap_loader_pbm.cpp

```diff
@@ -95,7 +95,7 @@ static bool _get_token(FileAccessRef &f, uint8_t &saved, PoolVector<uint8_t> &r_
 				resized = true;
 			}
 			if (resized) {
-				w = PoolVector<uint8_t>::Write();
+				PoolVector<uint8_t>::Write();
 				r_token.resize(token_max);
 				w = r_token.write();
```

Comment:
Please add a comment to mention why it's done this way then, to ensure we don't try to remove it again.